exporter/sumologicexporter/filtering_test.go

```diff
@@ -0,0 +1,99 @@
+package sumologicexporter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+func TestGetMetadata(t *testing.T) {
+	attributes := pdata.NewAttributeMap()
+	attributes.InsertString("key3", "value3")
+	attributes.InsertString("key1", "value1")
+	attributes.InsertString("key2", "value2")
+	attributes.InsertString("additional_key2", "value2")
+	attributes.InsertString("additional_key3", "value3")
+
+	regexes := []string{"^key[12]", "^key3"}
+	f, err := newFiltering(regexes)
+	assert.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

No point in continuing execution